app/views/bhf/table/column/_image.html.haml

```diff
@@ -1,2 +1,3 @@
 = content_tag link ? :a : :span, class: ('quick_edit' if add_quick_link), href: (link if link) do
-  = image_tag column_value, alt: column_value, class: 'preview_image'
\ No newline at end of file
+  - value = column_value.respond_to?(:midem) && column_value.midem || column_value
```

Comment:
but you have the `paperclip` field, that is also specific, `midem` is one standard image mapping rules for `carrierwave`